funfact/lang/_semiring.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from ._ast import Primitives as P
+from ._tsrex import TsrEx, EinopEx
+
+
+def minplus(lhs: TsrEx, rhs: TsrEx):
```

Comment:
This errors for me: 

`tsrex = minplus(A[i, j], B[j,k])` returns
`AttributeError: 'TsrEx' object has no attribute '_as_node'`

Reply:
The same error happens with the math operations in `_math.py` for which the `_as_node` function is used.